packages/adapter-node/src/handler.js

```diff
@@ -11,7 +11,9 @@ import { env } from './env.js';
 /* global ENV_PREFIX */
 
 const server = new Server(manifest);
-server.init({ env: process.env });
+// Dear Rich,
+// it says top-level await isn't available. Should it be?
+let init_promise = server.init({ env: process.env });
```

Comment:
I think we should update the tsconfig to allow TLA and see if anyone complains. it's supported in all the node versions we care about

```suggestion
await server.init({ env: process.env });
```